### DISCUSSION TITLE
Does this work for peleton app subscription workouts !

### DISCUSSION THREAD

**Discussion by SkyYogi-VSB** [philosowaffle/peloton-to-garmin#563] [General]
i asked Bard the same question but wanted to confirm .
i use the bike with my app subscription ...perhaps this will the reason to upgrade (Peleton's scheme  is successful) 
here is what Bard had to say
"No, the Peloton to Garmin (P2G) sync tool does not currently sync app workouts. This is because Peloton does not provide an API or other means for third-party tools to access app workout data. The P2G sync tool can only sync workouts that are recorded directly on a Peloton bike or treadmill."

**Comment by philosowaffle** [comment]
I don't think Bard is correct here. While I do have a Peloton bike, I've also taken Meditation and Stretching classes from the App on my phone and they synced just fine.  I also know others have used P2G to sync outdoor runs, rides, and walks recorded via the Peloton app on their phones.

**Reply by SkyYogi-VSB** [reply]
Thank you, perhaps i should have worded my question a bit more carefully !  i do have a bike but i use the app+ subscription to do my bike workouts vs the all access subscription  ; so i think its not about the app but just the subscription type one may have ; will be awesome if someone else with app+ subscription can verify this working for them; I think i have configured this correctly using github actions which runs successfully but there is no data to sync i believe (because of the subscription type i have)

**Reply by philosowaffle** [reply]
I took a peek at your Github action and based on the log output I think some configuration is missing.  Can you check that all of the [Secrets ](https://philosowaffle.github.io/peloton-to-garmin/install/github-action.html#secrets)have been set up in your repo?  There are four secrets (shown in that table in docs) that need to get added in order for P2G to run.  If you already have those secrets setup, do you mind just triple checking the spelling of them?

**Reply by SkyYogi-VSB** [reply]
yes i was indeed missing one ; added , ran and workouts ported right over; Sorry about all the fuss and thank you so much for creating this and engaging here ; much appreciated.
APP+ subscription workout do port anyone reading this in the future; Both me and Bard were wrong ....classic hallucination by AI and human both ...AI led the human
